src/js/plugin/sdk/actions/MarathonActions.js

```diff
@@ -0,0 +1,30 @@
+import MarathonService from "../services/MarathonService";
+
+var service = new MarathonService();
+
+export default class MarathonActions {
+
+  getDeployments() {
```

Comment:
Let's make these "actions" static so that we don't need to create and instance of `MarathonActions`.

Reply:
:+1: